content/en/docs/tasks/traffic-management/traffic-shifting/index.md

```diff
@@ -113,7 +113,7 @@ article [Canary Deployments using Istio](/blog/2017/0.1-canary/).
 
 1. Remove the application routing rules:
 
-    {{< text syntax=bash snip_id=cleanup >}}
+    {{< text syntax=bash snip_id="cleanup">}}
```

Comment:
Missing " " (space) character. UI and snippet generator both require code blocks to have space before `>}}`.

Reply:
oh yeah. Forgot to remove this. In my local, hugo complained about that and had to change it. Undoing this addition.